thoth/user_api/api_v1.py

```diff
@@ -80,9 +81,48 @@
 
 _PROVENANCE_CHECK_PROTECTED_FIELDS = frozenset({"kebechet_metadata"})
 
+
+# custom metric to expose cache hit rates for different service endpoints
+def _create_metrics_service_cache_hits(services: typing.List[str]):
+    service_request_cache_hit_rate: typing.Dict[str, dict] = {}
+
+    for service in services:
```

Comment:
Any reason to have this in a for loop rather than defining these metrics statically? I think we have 4 metrics.

Reply:
I didn't know if want to extend it to other in the future :) but yes I can make them statically :)